powerbi-docs/connect-data/desktop-report-lifecycle-datasets.md

```diff
@@ -109,7 +109,7 @@ When you use the Power BI service live connection, keep a few considerations and
 
 - Only users with Build permission for a dataset can connect to a published dataset by using the Power BI service live connection.
 - Free users only see datasets that are in their **My Workspace** and in Premium-based workspaces.
-- Because this connection is live, left navigation and modeling are disabled. You can connect to only one dataset in each report. The behavior is similar to a SQL Server Analysis Services (SSAS) connection. However, a new preview feature in Power BI now makes it possible to combine data from different sources. For more information, see [Use DirectQuery for Power BI datasets and Azure Analysis Services (preview)](../connect-data/desktop-directquery-datasets-azure-analysis-services.md).
+- Because this connection is live, left navigation and modeling are disabled. The behavior is similar to a SQL Server Analysis Services (SSAS) connection. However, composite models in Power BI makes it possible to combine data from different sources. For more information, see [Use composite models in Power BI Desktop](../connect-data/desktop-directquery-datasets-azure-analysis-services.md).
```

Comment:
```suggestion
- Because this connection is live, left navigation and modeling are disabled. The behavior is similar to a SQL Server Analysis Services (SSAS) connection. However, composite models in Power BI makes it possible to combine data from different sources. For more information, see [Use composite models in Power BI Desktop](../transform-model/desktop-composite-models.md).
```
Fixing link target to match article title